js/apps/admin-ui/src/user/UserGroups.tsx

```diff
@@ -74,7 +74,10 @@ export const UserGroups = ({ user }: UserGroupsProps) => {
     const indirect: GroupRepresentation[] = [];
     if (!isDirectMembership)
       joinedUserGroups.forEach((g) => {
-        const paths = g.path?.substring(1).split("/").slice(0, -1) || [];
+        const paths = (
```

Comment:
I think this code might need a comment for future reference to understand why this exists.